Gemfile.lock

```diff
@@ -159,7 +159,7 @@ GEM
       statsd-ruby (~> 1.4.0)
       unicorn (>= 5.4, < 5.8)
     govuk_document_types (0.9.2)
-    govuk_publishing_components (23.7.7)
+    govuk_publishing_components (23.9.0)
```

Comment:
It's worth rebasing this PR as this looks already commited to the repo.

I was going to suggest that the commit made a reference to this dependency change, but since it's already in the project it seems simpler to just rebase it away.

Reply:
Done